dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -940,9 +940,11 @@ private void addEmptyContainers(final IHTMLPage page,
 
                 Container container = null;
                 try {
-                    // this read path or id.
-                    container = liveMode ? this.getLiveContainerById(containerUUID.getIdentifier(), APILocator.systemUser(), template):
-                            this.getWorkingContainerById(containerUUID.getIdentifier(), APILocator.systemUser(), template);
+
+                    final Optional<Container> optionalContainer =
+                            APILocator.getContainerAPI().findContainer(containerUUID.getIdentifier(), APILocator.systemUser(), liveMode, false);
+
+                    container = optionalContainer.isPresent() ? optionalContainer.get() : null;
```

Comment:
Codacy found an issue: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)